Makefile

```diff
@@ -225,11 +228,11 @@ build-deb-arm64: dependencies-deb-arm64
 	@cd $(CURRENT_DIR)/axolotl-web && npm run build
 	@mkdir -p $(CURRENT_DIR)/build/linux-arm64/axolotl-web
 	@echo "Building (go)..."
-	@cd $(CURRENT_DIR) && env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl .
+	@cd $(CURRENT_DIR) && go build -o build/linux-arm64/axolotl .
 	@cp --recursive $(CURRENT_DIR)/axolotl-web/dist $(CURRENT_DIR)/build/linux-arm64/axolotl-web/
 	@cp --recursive $(CURRENT_DIR)/guis $(CURRENT_DIR)/build/linux-arm64/
 	@echo "Building (rust)..."
-	@cd $(CURRENT_DIR)/crayfish && cargo build --release
+	@cd $(CURRENT_DIR)/crayfish && $(RUST_BIN)/cargo build --release
```

Comment:
What about treating cargo like APT and WGET with its own path?

Reply:
Good idea, but since we have three binaries used from CARGO_PREFIX (rustup, cargo and cross), I think, it's sufficient to have that path as a variable and not cargo itself. Makes it a bit more consistent.